platform/admin/ui/src/main/webapp/js/views/installer/Certificate.view.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+/** Main view page for add. */
+define([
+    'marionette',
+    'icanhaz',
+    'underscore',
+    'backbone',
+    'jquery',
+    '../../models/CertsModel',
+    '../../models/FileHelper',
+    'text!templates/installer/certificate.handlebars',
+    'modelbinder',
+    'perfectscrollbar',
+    'multiselect'
+], function (Marionette, ich, _, Backbone, $, CertsModel, FileHelper, certificateTemplate) {
+
+    ich.addTemplate('certificateTemplate', certificateTemplate);
+
+    var CertificateView = Marionette.Layout.extend({
+        template: 'certificateTemplate',
+        model: new CertsModel(),
+        initialize: function () {
+            this.modelBinder = new Backbone.ModelBinder();
+        },
+        saveCerts: function () {
+            var view = this;
+            this.listenTo(this.model, 'invalid', function (model, errors) {
+                view.$('[name=keystorePass]').removeClass('error-border');
+                view.$('[name=keyPass]').removeClass('error-border');
+                view.$('[name=truststorePass]').removeClass('error-border');
+                errors.forEach(function (errorItem) {
+                    if (errorItem.name) {
+                        view.$('[name=' + errorItem.name + ']').addClass('error-border');
+                    } else if (errorItem.id) {
+                        view.$('[id=' + errorItem.id + ']').addClass('error-border');
+                    }
+                });
+            });
+
+            var saved = this.model.save();
+            if (saved) {
+                saved.success(function () {
+                    if (view.hasErrors()) {
+                        view.showCertErrors();
+                    }
+                });
+            }
+            return saved;
+        },
+        hasErrors: function () {
+            if (this.model.get('certErrors').length !== 0) {
+               return true;
+            }
+            return false;
+        },
+        showCertErrors: function () {
+                this.render();
+                this.$('.cert-errors').show();
```

Comment:
Rather than using jquery to do this, you could hook up a modelEvent to re-render the view if the model changes. Then in your template you could show this section if certErrors.length > 0. Then this entire function goes away.